### PR TITLE
Use mbstring functions without check if exist

### PR DESCRIPTION
**Pull request by davidglezz**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Since mbstring is a required extension, the function_exists('mb_xxx') check is always true. this is a continuation of https://github.com/PrestaShop/PrestaShop/pull/27359
| Type?             | refacto
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | CI green
| Possible impacts? | No

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27401)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/filemanager/include/utils.php [hunk 1]
```diff
@@ -142,13 +142,9 @@ function check_files_extensions_on_path($path, $ext)
 {
     if (!is_dir($path)) {
         $fileinfo = pathinfo($path);
-        if (function_exists('mb_strtolower')) {
             if (!in_array(mb_strtolower($fileinfo['extension']), $ext)) {
                 unlink($path);
-            } elseif (!in_array(Tools::strtolower($fileinfo['extension']), $ext)) {
-                unlink($path);
             }
```
Comment (atomiix) [inline admin-dev/filemanager/include/utils.php]: I think indentation is broken

**Comment by davidglezz** [comment]
@atomiix done

**Review by atomiix — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
@PrestaShop/prestashop-maintainers how should we test this PR?

**Comment by atomiix** [comment]
@matks I guess there's not so much way of testing it. Maybe we could ask QA team to run the full test suite but as long as the mbstring extension is present, I don't see why it wouldn't work?

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
I don't think it's useful to add tests or run the full CI for checking if the mbstring extension is properly loaded :thinking: 
I'm merging it as the CI is green and it's the requirement to merge this PR.

**Comment by PierreRambaud** [comment]
Thank you @davidglezz

**Pull request merged**